### PR TITLE
06 Weekly News Issue

### PR DESCRIPTION
**Pull request by garbas**
# Help

A substantial part of the effort of publishing the "weekly" news is collecting the news in the first place. If you see anything cool, want to share a job, meetup or presentation add a comment.

It's super helpful!

### CODE REVIEW

**Comment by garbas** [comment]
NixOS Security Advisory: Docker Local Privilege Escalation
http://lists.science.uu.nl/pipermail/nix-dev/2017-April/023329.html

**Comment by garbas** [comment]
Running Unpatched Binaries on NixOS
https://brianmckenna.org/blog/running_binaries_on_nixos

**Comment by garbas** [comment]
invite people to add themselves to planet.nixos.org
https://github.com/NixOS/nixos-org-configurations/blob/master/nixos-org/planet-feeds.nix

**Comment by grahamc** [comment]
> NixOS Security Advisory: Docker Local Privilege Escalation

This is a fun one because it is the first CVE taken against NixOS. A milestone!

**Comment by Profpatsch** [comment]
> This is a fun one because it is the first CVE taken against NixOS. A milestone!

Does this mean we get official CVE-numbers now?

**Comment by nicknovitski** [comment]
Compared to the other news, this will stand out in its triviality, but I found out I wasn't actually getting the benefits of vim-nix, and how to fix it:
http://nicknovitski.com/vim-nix-syntax

**Comment by garbas** [comment]
"NixOS, collection printemps-été 17" -> http://linuxfr.org/news/nixos-collection-printemps-ete-17

**Comment by garbas** [comment]
blog post recently using #11  and #ruby 
https://labs.uswitch.com/using-nix-to-manage-multiple-ruby-versions/

**Comment by garbas** [comment]
Mit alternativer Paketverwaltung: Linux-Distribution NixOS 17.03 erschienen
https://www.heise.de/ix/meldung/Mit-alternativer-Paketverwaltung-Linux-Distribution-NixOS-17-03-erschienen-3672948.html

**Comment by garbas** [comment]
"Using NixOS (As Your Main Desktop OS)" http://www.calebgossler.com/posts/using-nixos.html

**Comment by garbas** [comment]
Using nix (nixos) as a build system for Rust, and caching binary crates
https://nest.pijul.com/pmeunier/nix-rust
https://www.reddit.com/r/rust/comments/649h6m/using_nix_nixos_as_a_build_system_for_rust_and/

**Comment by garbas** [comment]
Augsburger Linux-Infotag
https://www.meetup.com/Munich-NixOS-Meetup/events/239077440/

**Comment by garbas** [comment]
Munich NixOS Meetup - Hackathon & Barbecue
https://www.meetup.com/Munich-NixOS-Meetup/events/239077247/?eventId=239077247

**Comment by svanderburg** [comment]
node2nix 1.2.0: https://www.npmjs.com/package/node2nix

**Comment by cillianderoiste** [comment]
Reproducible development environments with Nix: Johannes Bornhold, May 27, Munich https://pyconweb.com/talks/27-05-2017/reproducible-development-environments-with-nix

**Comment by garbas** [comment]
@shlevy: Takt has open-sourced a tool we're using to deploy our services from
hydra. Check out the blog post at
https://code.takt.com/announcing-hail-4da7208df56d !

**Comment by garbas** [comment]
new pypi2nix (v1.8.0) release -> https://github.com/garbas/pypi2nix/releases/tag/v1.8.0

**Comment by garbas** [comment]
everything above this line was added to 06 newsletter

**Comment by nbp** [comment]
`git bisect` can be used over 2 years of Nixpkgs/NixOS updates: https://github.com/lukego/blog/issues/17

**Comment by garbas** [comment]
"Install NixOS on Vultr" https://www.vultr.com/docs/install-nixos-on-vultr

File: content/2017/06-i-never-wished-a-man-dead.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+
+Events /Meetups
+===============
+
+- **Apr 22, 2017**, `Augsburger Linux-Infotag`_, Augsburg, Germany
+
+  The Augsburger Linux Infotag is a one-day conference with about 20 lectures
+  and 9 workshops on Linux, open source software and the digital society.
+  Admission is free.  We are there with a stand to NixOS represented :-)
```
Comment (globin) [inline content/2017/06-i-never-wished-a-man-dead.rst]: We are representing NixOS with a booth at the conference,

Comment (Profpatsch) [reply content/2017/06-i-never-wished-a-man-dead.rst]: Will you be there at LIT? Awesome!

**Comment by garbas** [comment]
i'm working on 06 right now and will release it in few minutes. interesting... this time we will be on time :)

**Pull request merged**